### ISSUE TITLE
CMA2

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/fmi-standard#428]
**Reported by edrenth on 11 Jan 2018 23:04 UTC**
With help of #422 the FMI co-simulation export can automatically insert an anti-aliasing and energy preserving continuous moving avergae filter; CMA2.

The CMA2 is part of the EPCE technology for non-iterative co-simulation of physical systems.  

Please read FCP_008_CMA2 for more information

----

Migrated-From: https://trac.fmi-standard.org/ticket/428

**Comment by modelica-trac-importer** [comment]
**Comment by edrenth on 12 Jan 2018 17:44 UTC**
Name change of FCP to avoid naming conflicts from WG numbers: FCP_008_CMA2 -> FCP_012_CMA2

**Comment by modelica-trac-importer** [comment]
**Comment by cbertsch on 1 Jun 2018 15:26 UTC**
To be discussed if this should be part of the FMI standard or if it is only a modeling or master tool issue.

**Comment by chrbertsch** [comment]
As the content of this proposal is not _necessaily_ needed to implement a future FMI  standard that would contain this  proposal, to my understanding of the CCLA there is no automatic granting of a patent license for a required patent  of Volvo in this case (as the CCLA does not apply for optional features) without an additional legally binding statement explicitly granting a patent license.

citing form the CCLA
> where such license applies only to those patent claims licensable by You that are _necessarily_ infringed by Your Contribution(s) alone or by combination of Your Contribution(s) with the Work to which such Contribution(s) were submitted.

**Comment by chrbertsch** [comment]
Closing for the following reasons: 
- CMA2 can be realized with Intermediate Variable Access that is already part of the master branch.
- It specifies a certain master algorithm (partially realized at FMU export). However, this issue tracker is about the FMI Standard -  describing only the _Interface_ So, @edrenth: If you have further proposals going beyond the Intermediate Variable Access on the interface: please open a separate ticket.
- as stated in the previous comment: there is a patent application pending on this topic, and submitting it from Volvo under the  CCLA is not sufficient to get a license granted (as this features is not necessary to implement the FMI standard, but touches only an  optional feature)

If someone disagrees, please reopen
